src/main/java/org/togetherjava/tjbot/commands/ICommand.java

```diff
@@ -0,0 +1,62 @@
+package org.togetherjava.tjbot.commands;
+
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.interactions.commands.build.CommandData;
+import org.togetherjava.tjbot.commands.example.CommandExample;
+
+/**
+ * The class all commands have to abstract.
+ * <br> Look at {@link CommandExample} for an example.
+ * <br> A second example will come for sub-commands.
+ *
+ */
+public interface ICommand {
```

Comment:
```suggestion
public interface Command {
```
Starting interfaces with I is an old standard that I don't think we need to follow. Feel free to disregard if the group feels otherwise.

Reply:
We don't need to follow it yeah, following it has no downsides.
While it does have the upside of telling the user it's a interface.
Which is nice, since commands have to implement it, not extend it.

Reply:
We should anyway not make changes for such minor things, but I thing we should stick with a naming convention throughout the project.

Reply:
I'd also suggest sticking to common _Java_ conventions. IIRC using I and comparable as prefix is more a thing for C-based languages.

Reply:
There's no Java convention telling you to not use or or use it ¯\_(ツ)_/¯
> Interface names should be capitalized like class names.
So by adding an I to the name, we are following Java conventions.

Reply:
Well yes, everything falls under that classification. Just saying that I've yet to see anyone code Java like that in the industry/higher education

Reply:
I don't really care so ig I'll change it